app/bundles/FormBundle/Entity/SubmissionRepository.php

```diff
@@ -89,11 +99,14 @@ function ($value) {
         $this->buildLimiterClauses($dq, $args);
 
         $dq->resetQueryPart('select');
-        $fieldAliasSql = ',';
-        foreach ($fieldAliases as $fieldAlias) {
-            $fieldAliasSql .= 'r.`'.$fieldAlias.'`,';
-        }
-        $fieldAliasSql = substr($fieldAliasSql, 0, -1);
+
+        $databasePlatform = $this->_em->getConnection()->getDatabasePlatform();
+        // Quote reserved keywords in field aliases
+        $fieldAliases = array_map(function ($alias) use ($databasePlatform) {
+            return $databasePlatform->quoteIdentifier($alias);
+        }, $fieldAliases);
```

Comment:
This could be simplified
```suggestion
        $fieldAliases = array_map(fn ($alias) => $databasePlatform->quoteIdentifier($alias), $fieldAliases);
```